.github/workflows/molecule-docker.yaml

```diff
@@ -17,7 +17,7 @@ concurrency:
 
 jobs:
   molecule-docker:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
we run into errors running sudo commands on Rocky 9 containers if we use `ubuntu-latest`